src/hdmf/spec/namespace.py

```diff
@@ -83,13 +85,16 @@ def contact(self):
 
     @property
     def author(self):
-        """String or list of strings with the authors or  None"""
+        """String or list of strings with the authors or None"""
         return self.get('author', None)
 
     @property
     def version(self):
-        """String, list, or tuple with the version or None """
-        return self.get('version', None)
+        """
+        String, list, or tuple with the version or SpecNamespace.UNVERSIONED
+        if the version is missing or empty
+        """
+        return self.get('version', None) or SpecNamespace.UNVERSIONED
```

Comment:
This would return a bool. Did you mean 

`return self.get('verzion', SpecNamespace.UNVERSIONED)`